templates/shared/combomarkdowneditor.tmpl

```diff
@@ -16,27 +16,27 @@ Template Attributes:
 	<div class="ui tab active" data-tab-panel="markdown-writer">
 		<markdown-toolbar class="gt-df">
```

Comment:
@silverwind 
> If you make the parent a flexbox (gt-df), the whitespace between tags will not matter anymore.

This is already the case, and yet the whitespace mattered…

Reply:
Hmm can't be as per https://www.w3.org/TR/css-flexbox-1/#flex-items

> if the entire sequence of child [text runs](https://www.w3.org/TR/css-display-3/#css-text-run) contains only [white space](https://www.w3.org/TR/CSS2/text.html#white-space-prop) (i.e. characters that can be affected by the [white-space](https://www.w3.org/TR/css-text-3/#propdef-white-space) property) it is instead not rendered (just as if its [text nodes](https://www.w3.org/TR/css-display-3/#css-text-node) were display:none).

I guess I need to test it.

Reply:
Have you tried to make `markdown-toolbar-group` have `display: inline-flex`?

Reply:
Not sure what the whitespce problem was but see my suggestion below, it looks alright to me.

Reply:
If I understand correctly, delvh wants to remove the spaces between the toolbar buttons.

```html
<markdown-toolbar class="gt-df">
	<div class="markdown-toolbar-group">
			<md-bold class="markdown-toolbar-button">...</md-bold><!-- SPACE -->
	</div>
</markdown-toolbar>
```

To hide the SPACE, its parent element should have `display: flex/inline-flex`.

The question in the comment: the `<markdown-toolbar class="gt-df">` only affects its direct children: `<div class="markdown-toolbar-group">`, so the SPACE is still there.

Reply:
Indeed, flexbox space removal only affects direct children.